mottak/src/test/kotlin/no/nav/melosys/soknadmottak/soknad/SoknadStatsCacheTest.kt

```diff
@@ -8,10 +8,12 @@ import io.mockk.impl.annotations.MockK
 import io.mockk.junit5.MockKExtension
 import no.nav.melosys.soknadmottak.config.MetrikkConfig
 import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.ExtendWith
 
 @ExtendWith(MockKExtension::class)
+@Disabled("Alt for ustabile tester")
```

Comment:
Hvis det stemmer så bør vi bruke en Spring cache her i stedet for noe hjemmelaget